.github/workflows/pr_scan.yml

```diff
@@ -23,7 +26,7 @@ jobs:
     - name: Select latest Xcode
       uses: maxim-lobanov/setup-xcode@v1
       with:
-        xcode-version: latest-stable
+        xcode-version: '16.1'
```

Comment:
Why do we need to lock it now?

Reply:
We want to sync it with our local machines (like with v5) so the tests are reproducable